03-wrangling.Rmd

```diff
@@ -905,7 +905,13 @@ if(knitr:::is_latex_output()){
 
 **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Let's now put your newly acquired data wrangling skills to the test! 
 
-An airline industry measure of a passenger airline's capacity is the [available seat miles](https://en.wikipedia.org/wiki/Available_seat_miles), which is equal to the number of seats available multiplied by the number of miles or kilometers flown summed over all flights. So, for example, say an airline had 2 flights using a plane with 10 seats that flew 500 miles and 3 flights using a plane with 20 seats that flew 1000 miles, the available seat miles would be 2 $\times$ 10 $\times$ 500 $+$ 3 $\times$ 20 $\times$ 1000 = 70,000 seat miles.
+An airline industry measure of a passenger airline's capacity is the [available seat miles](https://en.wikipedia.org/wiki/Available_seat_miles), which is equal to the number of seats available multiplied by the number of miles or kilometers flown summed over all flights. So, for example, say an airline had 2 flights using a plane with 10 seats that flew 500 miles and 3 flights using a plane with 20 seats that flew 1000 miles, the available seat miles would be 2 $\times$ 10 $\times$ 500 $+$ 3 $\times$ 20 $\times$ 1000 = 70,000 seat miles. 
+
+For example, let's consider the scenario in Figure \@ref(fig:available-seat-miles). Since the airplanes has 4 seats and it travels 200 miles, the available seat miles are 4 $\times$ 200 = 800.
+
+```{r available-seat-miles, echo=FALSE, fig.cap="Example of available seat miles for one flight.", fig.width=8, purl=FALSE}
+knitr::include_graphics("images/flowcharts/flowchart/flowchart.012.png")
```

Comment:
I like it! That's helpful.

Reply:
I'll merge this in and then sync it back with the `crc-edits` branch where I've been getting the images to appear in the right spots.